feat: add footer

dyeresko opened the pull request:
1. Task: [link](https://github.com/rolling-scopes-school/tasks/blob/master/react/modules/tasks/final.md)
2. Screenshot:
3. Deploy: [link](https://github.com/)
4. Done 02.09.2025 / deadline 22.09.2025
5. Score: 5 / 500
- [x] Footer contains general information about developers with year of creation and logo with link to the school. - 5 points

dyeresko committed:
feat: add footer

novogran commented on src/components/ui/dialog.tsx:
Почему dialog должен быть клиентским?

novogran commented on src/lib/utils.ts:
Разве не стоит сделать вместо патки lib, папку utils и в ней файл с с названием tailwind-utils.ts например?

dyeresko replied:
Папка либ автоматически создалась

novogran commented on src/types/interfaces.ts:
Стоит переименовать файл в developerInfo.types.ts

novogran reviewed (changes requested):
1. Добавить тесты
2. Стоит добавить для изображений в next.config.ts
`images: {
    unoptimized: true,
  },`
 3. Нет поддержки мультиязычности

dyeresko committed:
fix: fix issues that occured during code review

novogran committed:
feat: add not found route

novogran committed:
feat: add tests for not found route

dyeresko committed:
test: test footer and dependent components

novogran commented on src/data/developerData.ts:
```diff
@@ -0,0 +1,28 @@
+    id: 1,
+    name: 'Daniil Sudorgin',
+    // TODO: need to replace description to something more appropriate
+    description: 'some text',
```
TODO не должно быть. Нужно заполнить всю инфу.

Zilusion replied:
```suggestion
    description: 'A front-end developer with commercial experience in Svelte, who used this course to deeply master the modern React ecosystem. He focuses on building robust, scalable applications with a strong emphasis on clean architecture, full test coverage, and pragmatic engineering solutions.',
```

novogran reviewed (changes requested):
1. Не должно быть TODO, перед мержем дожно быть сделано всё
2. Добавить мультиязычность

novogran committed:
refactor: add shadcn as a child for Link

novogran committed:
refactor: change NotFound test file name

novogran committed:
refactor: delete old test file

novogran committed:
refactor: change test description

novogran committed:
refactor: change tests description in english

novogran committed:
Merge pull request #17 from novogran/not_found_route

Not found route

Zilusion committed:
feat: implement internationalization

* chore: configure development tooling

* feat: implement internationalization

Implements full internationalization support for the application,
enabling multi-language capabilities as required by the project specification.
- Integrates `next-intl` library for locale-based
routing, message formatting, and translations.
- Adds support for English (en) and Russian (ru)
languages with corresponding message files.
- Restructures the application file layout to use
the `[locale]` dynamic segment in the App Router.
- Creates `LanguageSwitcher` UI component.
- Includes tests for the `LanguageSwitcher`
to ensure its functionality.

* refactor: rewrite language switcher with shadcn/ui and add tests

dyeresko committed:
Merge remote-tracking branch 'origin/footer' into develop

git add .# Please enter the commit message for your changes. Lines starting

dyeresko committed:
feat: add translations to footer

novogran reviewed (approved):
Отличная работа

Zilusion reviewed (approved):
Отлично!
